test/fuzz/basic.ml

```diff
@@ -13,6 +13,17 @@ let val_type =
   let+ num_type in
   Num_type num_type
 
+let symbolic =
+  let+ str = bytes in
+  (Symbolic str : indice)
+
+let raw =
+  let+ i = int in
+  (Raw i : indice)
+
+let indice =
```

Comment:
i don't think we're ever going to really need any of these three functions. You should remove them. (I explain a little bit more below).